bin/create-orders.ts

```diff
@@ -0,0 +1,173 @@
+import assert from 'assert';
+import { default as dayjs } from 'dayjs';
+import * as dotenv from 'dotenv';
+import _ from 'lodash';
+import PQueue from 'p-queue';
+import pRetry from 'p-retry';
+import { default as Shopify } from 'shopify-api-node';
+
+import { MinimalOrder, MinimalOrderDto } from '../src/shopify-types.js';
+import {
+    isStringArray,
+    parseStringToIntUndefined,
+    undefinedStringToJsonStringArray,
+} from '../src/type-utilities.js';
+
+export type Configuration = {
+    shopUrl: string;
+    shopAccessToken: string;
+    numOrders: number;
+    skus: string[];
+};
+
+function isValidConfig(
+    configuration: Partial<Configuration>,
+): configuration is Configuration {
+    if (!Object.values(configuration).every((entry) => entry !== undefined)) {
+        return false;
+    }
+
+    return isStringArray(configuration.skus);
+}
+
+function increaseArraySize<T>(array: T[], nDesired: number): T[] {
+    const requiredDuplications = Math.ceil(nDesired / array.length);
+    for (let expansion = 1; expansion < requiredDuplications; expansion += 1) {
+        array = array.concat(array);
+    }
+
+    return _.take(array, nDesired);
+}
+
+async function callShopifyOrderApi(
```

Comment:
"create" is the more descriptive verb here:

```suggestion
async function createOrdersViaShopifyAdminApi(
```